src/App/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+
+type Props = {
+    children?: any,
+}
+
+export const ErrorBoundary: React.FC<Props> = (props: Props) => {
+    const { children } = props
+
+    const ErrorMessage = () => (
+        <h2>
+            We are sorry, The Roulette just stops :( 
+        </h2>
+    )
```

Comment:
also, it's ok to assign a value to variable as a component:

```suggestion
    const errorMessage = (
        <h2>
            We are sorry, The Roulette just stops :( 
        </h2>
    )
```

and then use it like this in return statement: 
```
return error ? errorMessage : children
```

Reply:
implemented 🚀